mindsdb/integrations/handlers/s3_handler/s3_handler.py

```diff
@@ -125,6 +137,8 @@ def _connect_duckdb(self) -> DuckDBPyConnection:
         """
         # Connect to S3 via DuckDB.
         duckdb_conn = duckdb.connect(":memory:")
+        duckdb_conn.execute("INSTALL httpfs")
+        duckdb_conn.execute("LOAD httpfs")
```

Comment:
@MinuraPunchihewa 
a question about this: should we keep it?
I removed it in previous PR, but yesterday Mauricio said he had error on docker version. 
Today I tested docker version and it worked for me (it is latest release without these lines )

Reply:
@ea-rus When I was working on this, I remember that I could not get this to work without these two lines. I think it would be best to keep them. If I am not mistaken, they do not take any time at all to execute either.

Reply:
ok, let's keep it